scripts/travis-run.sh

```diff
@@ -1,6 +1,16 @@
 #!/bin/bash
 set -euo pipefail
 
+if [[ $TRAVIS_BRANCH != "master" && $TRAVIS_PULL_REQUEST == "false" && $TRAVIS_REPO_SLUG == "bioconda/bioconda-recipes" ]]
+then
+    echo ""
+    echo "Tests are skipped for pushes to the main bioconda-recipes repo."
+    echo "If you have opened a pull request, please see the full tests for that PR."
+    echo "See https://bioconda.github.io/build-system.html for details"
+    echo ""
+    exit 0
+fi
+
```

Comment:
Yes, that's right. We can't merge if one fails. I wonder if all this is worth it. Are there really people that patiently wait until their branch is build in the fork until they issue a PR?